tests: bsim: bluetooth: audio build failure

**Describe the bug**
tests: bsim: bluetooth: audio build failure
The variable `sem_started` is using for `k_sem_take` in function `test_sink_encrypted_incorrect_code` undeclared 

**To Reproduce**
run cmd to build test:`nice tests/bsim/bluetooth/compile.sh`


**Expected behavior**
This test build passed

**Impact**
Cause CI to fail, preventing the merging of PRs

**Logs and console output**
See #78073 CI bsim checks
```
Run tests/bsim/ci.bt.sh
Run '/home/runner/k8s/index.js'
+ nice tests/bsim/bluetooth/compile.sh
Building /opt/bsim//bin/bs_nrf52_bsim_native_tests_bsim_bluetooth_audio_prj_conf
Building /opt/bsim//bin/bs_nrf52_bsim_native_samples_bluetooth_bap_broadcast_source_prj_conf
Building /opt/bsim//bin/bs_nrf52_bsim_native_tests_bsim_bluetooth_audio_samples_bap_broadcast_sink_prj_conf
Building /opt/bsim//bin/bs_nrf52_bsim_native_samples_bluetooth_bap_unicast_server_prj_conf
Building /opt/bsim//bin/bs_nrf52_bsim_native_tests_bsim_bluetooth_audio_samples_bap_unicast_client_prj_conf
Building /opt/bsim//bin/bs_nrf52_bsim_native_tests_bsim_bluetooth_audio_samples_cap_initiator_broadcast_prj_conf
Building /opt/bsim//bin/bs_nrf52_bsim_native_tests_bsim_bluetooth_audio_samples_cap_initiator_unicast_prj_conf
Building /opt/bsim//bin/bs_nrf52_bsim_native_tests_bsim_bluetooth_audio_samples_cap_acceptor_broadcast_prj_conf
Building /opt/bsim//bin/bs_nrf52_bsim_native_tests_bsim_bluetooth_audio_samples_cap_acceptor_unicast_prj_conf
Failure building tests/bsim/bluetooth/audio prj.conf for nrf52_bsim/native
  You can rebuild it with
  'cmake' '-GNinja' '-DBOARD_ROOT=/__w/zephyr/zephyr' '-DBOARD=nrf52_bsim/native' '-DOVERLAY_CONFIG=overlay-bt_ll_sw_split.conf' '-DEXTRA_CONF_FILE=' '-DSNIPPET=' '-DCONFIG_COVERAGE=y' '-DCMAKE_EXPORT_COMPILE_COMMANDS=ON' '-DCONFIG_COMPILER_WARNINGS_AS_ERRORS=y' '-DCONFIG_ASSERT=y' '' '-DAPP_DIR=/__w/zephyr/zephyr/tests/bsim/bluetooth/audio' '/__w/zephyr/zephyr/share/sysbuild/' && ninja 
[1/9] Creating directories for 'audio'
[2/9] No download step for 'audio'
[3/9] No update step for 'audio'
[4/9] No patch step for 'audio'
[5/9] No configure step for 'audio'
[5/9] Performing build step for 'audio'
[1/451] Preparing syscall dependency handling
[2/451] Generating include/generated/zephyr/version.h
-- Zephyr version: 4.0.99 (/__w/zephyr/zephyr), build: v4.0.0-2738-g33ab075dd4d6
[3/451] Generating misc/generated/syscalls_subdirs.trigger
[4/451] Generating misc/generated/syscalls.json, misc/generated/struct_tags.json
[5/451] Generating include/generated/device-api-sections.ld, include/generated/device-api-sections.cmake
[6/451] Generating include/generated/zephyr/syscall_dispatch.c, include/generated/zephyr/syscall_exports_llext.c, syscall_weakdefs_llext.c, include/generated/zephyr/syscall_list.h
[7/451] Generating include/generated/zephyr/kobj-types-enum.h, include/generated/zephyr/otype-to-str.h, include/generated/zephyr/otype-to-size.h
[8/451] Generating include/generated/zephyr/driver-validation.h
[9/451] Building C object zephyr/CMakeFiles/offsets.dir/arch/posix/core/offsets/offsets.c.obj
[[10](https://github.com/zephyrproject-rtos/zephyr/actions/runs/12498462595/job/34872270421?pr=78073#step:13:11)/451] Generating include/generated/zephyr/offsets.h
[11/451] Building C object CMakeFiles/app.dir/src/bap_unicast_server_test.c.obj
[[12](https://github.com/zephyrproject-rtos/zephyr/actions/runs/12498462595/job/34872270421?pr=78073#step:13:14)/451] Building C object CMakeFiles/app.dir/src/bap_common.c.obj
[[13](https://github.com/zephyrproject-rtos/zephyr/actions/runs/12498462595/job/34872270421?pr=78073#step:13:15)/451] Building C object CMakeFiles/app.dir/src/bap_bass_broadcaster_test.c.obj
[[14](https://github.com/zephyrproject-rtos/zephyr/actions/runs/12498462595/job/34872270421?pr=78073#step:13:16)/451] Building C object CMakeFiles/app.dir/src/bap_broadcast_sink_test.c.obj
FAILED: CMakeFiles/app.dir/src/bap_broadcast_sink_test.c.obj 
ccache /usr/bin/gcc -DKERNEL -DK_HEAP_MEM_POOL_SIZE=0 -DMBEDTLS_CONFIG_FILE=\"config-tls-generic.h\" -DNRF52833_XXAA -D__ZEPHYR__=1 -I/__w/zephyr/modules/bsim_hw_models/nrf_hw_models/src/nrfx/mdk_replacements -I/__w/zephyr/modules/bsim_hw_models/nrf_hw_models/src/nrfx/hal_replacements -I/__w/zephyr/zephyr/bsim_out/tests/bsim/bluetooth/audio/bs_nrf52_bsim_native_tests_bsim_bluetooth_audio_prj_conf/audio/zephyr/include/generated/zephyr -I/__w/zephyr/zephyr/include -I/__w/zephyr/zephyr/bsim_out/tests/bsim/bluetooth/audio/bs_nrf52_bsim_native_tests_bsim_bluetooth_audio_prj_conf/audio/zephyr/include/generated -I/__w/zephyr/zephyr/soc/native/inf_clock -I/__w/zephyr/zephyr/boards/native/nrf_bsim -I/__w/zephyr/zephyr/boards/native/nrf_bsim/soc -I/__w/zephyr/zephyr/boards/native/nrf_bsim/common -I/__w/zephyr/zephyr/boards/native/nrf_bsim/common/cmsis -I/__w/zephyr/zephyr/scripts/native_simulator/common/src/include -I/__w/zephyr/zephyr/soc/nordic/common -I/__w/zephyr/zephyr/subsys/testsuite/include -I/__w/zephyr/zephyr
/__w/zephyr/zephyr/tests/bsim/bluetooth/audio/src/bap_broadcast_sink_test.c: In function ‘test_sink_encrypted_incorrect_code’:
/__w/zephyr/zephyr/tests/bsim/bluetooth/audio/src/bap_broadcast_sink_test.c:1090:29: error: ‘sem_started’ undeclared (first use in this function); did you mean ‘pm_state’?
 1090 |                 k_sem_take(&sem_started, K_FOREVER);
      |                             ^~~~~~~~~~~
      |                             pm_state
/__w/zephyr/zephyr/tests/bsim/bluetooth/audio/src/bap_broadcast_sink_test.c:1090:29: note: each undeclared identifier is reported only once for each function it appears in
[[15](https://github.com/zephyrproject-rtos/zephyr/actions/runs/12498462595/job/34872270421?pr=78073#step:13:17)/451] Building C object CMakeFiles/app.dir/src/bap_stream_tx.c.obj
[16/451] Building C object CMakeFiles/app.dir/src/cap_initiator_unicast_test.c.obj
[17/451] Building C object CMakeFiles/app.dir/src/bap_unicast_client_test.c.obj
[18/451] Building C object CMakeFiles/app.dir/src/cap_commander_test.c.obj
[19/451] Building C object CMakeFiles/app.dir/src/csip_notify_client_test.c.obj
[20/451] Building C object CMakeFiles/app.dir/src/bap_stream_rx.c.obj
[21/451] Building C object CMakeFiles/app.dir/src/gmap_ugt_test.c.obj
[22/451] Building C object CMakeFiles/app.dir/src/common.c.obj
[23/451] Building C object CMakeFiles/app.dir/src/bap_broadcast_source_test.c.obj
[24/451] Building C object CMakeFiles/app.dir/src/csip_set_member_test.c.obj
[25/451] Building C object CMakeFiles/app.dir/src/bap_broadcast_assistant_test.c.obj
[26/451] Building C object CMakeFiles/app.dir/src/csip_set_coordinator_test.c.obj
[27/451] Building C object CMakeFiles/app.dir/src/cap_initiator_broadcast_test.c.obj
[28/451] Building C object CMakeFiles/app.dir/src/bap_scan_delegator_test.c.obj
[29/451] Building C object CMakeFiles/app.dir/src/cap_acceptor_test.c.obj
[30/451] Building C object CMakeFiles/app.dir/src/csip_notify_server_test.c.obj
[31/451] Building C object CMakeFiles/app.dir/src/gmap_ugg_test.c.obj
ninja: build stopped: subcommand failed.
FAILED: _sysbuild/sysbuild/images/audio-prefix/src/audio-stamp/audio-build
```

**Environment (please complete the following information):**

 - OS: linux
 - Toolchain Zephyr SDK
 - Commit SHA or Version used: c5fc9a2b173e49352812539f3612bafdd5d9bc46 (submitted by PR #77642)

**Additional context**
<!--
Add any other context that could be relevant to your issue, such as pin setting,
target configuration, ...
-->

--- a/tests/bsim/bluetooth/audio/src/bap_broadcast_sink_test.c
+++ b/tests/bsim/bluetooth/audio/src/bap_broadcast_sink_test.c
@@ -1087,7 +1087,7 @@ static void test_sink_encrypted_incorrect_code(void)
 	/* Wait for all to be started */
 	printk("Waiting for streams to be started\n");
 	for (size_t i = 0U; i < ARRAY_SIZE(streams); i++) {
-		k_sem_take(&sem_started, K_FOREVER);
+		k_sem_take(&sem_stream_started, K_FOREVER);
 	}
 
 	printk("Waiting for data\n");

